doc/docs/troubleshooting.md

```diff
@@ -28,4 +28,34 @@ sudo usermod -a -G dialout <username>
 
 ## nRF52840 USB Problems
 
-If you receive errors related to the nRF52840 Dongle USB connection on Windows, see [GitHub](https://github.com/NordicSemiconductor/pc-nrfconnect-launcher/blob/master/doc/win32-usb-troubleshoot.md).
+If you receive errors related to the nRF52840 Dongle USB connection on Windows, see [GitHub](https://github.com/NordicSemiconductor/pc-nrfconnect-launcher/blob/main/doc/non_mkdocs/win32-usb-troubleshoot.md).
+
+## Programming with J-Link does not work
+
+If you select a device that uses the nRF5340 SoC as the interface MCU and attempt to program it, you might get the following entries or similar in the log:
+
+```
+Uploading image through JLink: 0%
+Device programming completed.
+Device setup completed
+Getting information from J-Link debugger...
+Found device type: unknown. J-Link firmware: J-Link OB-nRF5340-NordicSemi compiled Nov 7 2022 16:22:01.
+```
+
+This issue is related to readback protection of the nRF5340 MCU.
+To solve the issue, complete the following steps:
+
+1. Make sure you have nrfjprog installed (part of the [nRF Command Line Tools](https://www.nordicsemi.com/Products/Development-tools/nrf-command-line-tools/download#infotabs)).
+1. Open a command line terminal.
+1. Connect the DK to the serial port.
+1. Run the following command:
+
+    ```
+    nrfjprog --recover
+    ```
+
+1. Disconnect and connect the DK to the serial port again.
+1. [Select the device](./overview_and_ui.md#select-device).
+1. When prompted about programming, select **Yes**.
+
+The app will program the DK and the connection will successfully open with a related log entry.
```

Comment:
@datenreisender , @chunfantasy , please advise if we want this documented. Relates to https://nordicsemi.atlassian.net/browse/NRFFOSDK-13815

Reply:
Even though the app does not officially support devices with the nRF5340 I think it is a good idea to mention this here. I didn't check whether the described procedure works though. :-)